gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -14,7 +14,9 @@ <h1 class="title">A distributed vulnerability database for Open Source</h1>
         <a class="cta-primary link-button" href="{{ url_for('frontend_handlers.list_vulnerabilities') }}">Search
           Vulnerability Database</a>
         <a class="cta-primary link-button" href="#use-the-api">Use the API</a>
-        <a class="cta-primary link-button" href="#use-the-cli">CLI Tools</a>
+        <a class="cta-primary link-button" href="#use-vulnerability-scanner">Vulnerability Scanner</a>
```

Comment:
These buttons are now all getting a bit crowded by being all on the same row.

Can we move the buttons starting from "Vulnerability Scanner" into a separate row?

I.e.

| Search Vulnerability Database | Use the API |
| Vulnerability Scanner | Remediation Tools | GitHub Workflows |

Reply:
Update via 159d4dc . Now it looks like:

<img width="1171" alt="Screenshot 2024-05-09 at 6 56 12 PM" src="https://github.com/google/osv.dev/assets/73332835/18d8f7e4-7906-4223-8480-1ef9fade3865">
<img width="742" alt="Screenshot 2024-05-09 at 6 55 55 PM" src="https://github.com/google/osv.dev/assets/73332835/448cb64f-87d8-4b69-875b-55dd490c70c0">

Reply:
For me, it looks like there's a lack of spacing in between the buttons:

![image](https://github.com/google/osv.dev/assets/759062/75e13ec2-cb61-468d-8f66-eb67f8fb3723)

Reply:
My bad, please ignore. I think my local CSS was stale somehow.